bin/BootstrapTypes/Globals.cpp

```diff
@@ -1879,6 +1913,21 @@ std::map<std::pair<std::string, std::string>, std::string> kConditionalNullptr{
    "  if (self.isResultDependent()) {\n"
    "    return std::nullopt;\n"
    "  }\n"},
+  {{"PointerAttr", "DerefType"},
+   "  if (!self.getDerefTypeLoc()) {\n"
+   "    return std::nullopt;\n"
+   "  }\n"},
+  {{"OwnerAttr", "DerefType"},
```

Comment:
I thought we don't try to get `Type`s from `TypeLoc`s anymore?

Reply:
We do get it in some cases and generate the pasta type.